XLS-0074d-amm-clawback/README.md

```diff
@@ -30,8 +30,37 @@ Currently, accounts that have enabled clawback cannot create AMM pools. This pro
 ## 2. Specification
 ### 2.1. AMM and Frozen Asset 
 #### 2.1.1. Prohibiting depositing new tokens
+
+The AMMClawback amendment introduces changes to the behavior of the `AMMDeposit` transaction when tokens in trustlines interact with Automated Market Maker (AMM) pools. 
+
+Assume we have created an Automated Market Maker (AMM) pool with two assets: A and B. Currently, asset A is frozen by individual freeze. The following table outlines whether specific scenarios are allowed or prohibited before and after the amendment:
```

Comment:
its better to address the changes as "current behavior" vs "proposed behavior", since the spec if a proposal, not a amendment yet.